drivers/overlay/overlay.go

```diff
@@ -264,7 +260,7 @@ func parseOptions(options []string) (*overlayOptions, error) {
 	return o, nil
 }
 
-func supportsOverlay() error {
+func SupportsOverlay() error {
```

Comment:
Is this actually called anywhere else?  I'm not seeing references to it elsewhere.  Just wondering if we should drop it entirely or if we can do the check when not in a container.